index.js

```diff
@@ -0,0 +1,110 @@
+const express = require('express');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs');
+
+const db = require('./data/dbConfig');
+
+const server = express();
+const PORT = 4200;
+
+const secret = "SuperCaliFragilisticExpialadocious8675309"; //heh, two fun old references here! :-)
```

Comment:
Try to use a .env file so that the password is secure and not accessible in the repository